lib/seed/protocol_seeder.rb

```diff
@@ -0,0 +1,49 @@
+require_dependency "seed/config"
+
+module Seed
+  class ProtocolSeeder
+    include ConsoleLogger
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def initialize(config:)
+      @config = config
+      @logger = Rails.logger.child(class: self.class.name)
+      announce "Starting #{self.class} with #{config.type} configuration"
+    end
+
+    attr_reader :config
+    attr_reader :logger
+
+    delegate :stdout, to: :config
+
+    PROTOCOL_DRUGS = [
+      {name: "Amlodipine", dosage: "5 mg", rxnorm_code: "329528", drug_category: :hypertension_ccb, stock_tracked: true},
+      {name: "Amlodipine", dosage: "10 mg", rxnorm_code: "329526", drug_category: :hypertension_ccb, stock_tracked: true},
+      {name: "Telmisartan", dosage: "40 mg", rxnorm_code: "316764", drug_category: :hypertension_arb, stock_tracked: true},
+      {name: "Telmisartan", dosage: "80 mg", rxnorm_code: "316765", drug_category: :hypertension_arb, stock_tracked: true},
+      {name: "Losartan", dosage: "50 mg", rxnorm_code: "979467", drug_category: :hypertension_arb, stock_tracked: true},
+      {name: "Hydrochlorothiazide", dosage: "12.5 mg", rxnorm_code: "316047", drug_category: :hypertension_diuretic, stock_tracked: true},
+      {name: "Hydrochlorothiazide", dosage: "25 mg", rxnorm_code: "316049", drug_category: :hypertension_diuretic, stock_tracked: true},
+      {name: "Chlorthalidone", dosage: "12.5 mg", rxnorm_code: "331132", drug_category: :hypertension_diuretic, stock_tracked: true}
```

Comment:
Do you think these will remain pretty static over time? 

I'm wondering if we should extract to YAML or something if that happens ...lots of magic numbers / names here.

Reply:
They should be pretty static, the medication protocols in India are fairly solid. I'd be ok leaving as is.

Reply:
These should be pretty static over years, but it's probably still worth extracting them into a YAML so it's a declarative reference for us through time.